pkg/adapter/v2/cloudevents.go

```diff
@@ -168,21 +173,28 @@ func (c *client) reportCount(ctx context.Context, event cloudevents.Event, resul
 
 		var res *http.Result
 		if !cloudevents.ResultAs(result, &res) {
-			return c.reportError(reportArgs, result)
+			c.reportError(reportArgs, result)
 		}
 
 		if rErr := c.reporter.ReportEventCount(reportArgs, res.StatusCode); rErr != nil {
-			// metrics is not important enough to return an error if it is setup wrong.
-			// So combine reporter error with ce error if not nil.
-			if result != nil {
-				result = fmt.Errorf("%w\nmetrics reporter error: %s", result, rErr)
+			fmt.Printf("metrics reporter error %s: %s \n", result, rErr)
```

Comment:
These still seem like they are not doing anything since we don't use them anywhere?